features/fixtures/statistics.json

```diff
@@ -0,0 +1,116 @@
+{
+  "analytics_identifier":null,
+  "base_path":"/research-and-statistics",
+  "content_id":"8f827d53-9ad1-4b90-b6ae-2301c1ecdf02",
+  "content_purpose_document_supertype":"navigation",
+  "document_type":"finder",
+  "email_document_supertype":"other",
+  "first_published_at":"2019-01-15T14:47:48.000+00:00",
+  "government_document_supertype":"other",
+  "locale":"en",
+  "navigation_document_supertype":"other",
+  "phase":"alpha",
+  "public_updated_at":"2019-01-15T14:47:48.000+00:00",
+  "publishing_app":"finder-frontend",
+  "rendering_app":"finder-frontend",
+  "schema_name":"finder",
+  "search_user_need_document_supertype":"government",
+  "title":"Research and statistics",
+  "updated_at":"2019-01-15T14:47:48.000+00:00",
+  "user_journey_document_supertype":"finding",
+  "withdrawn_notice":{},
+  "publishing_request_id":"",
+  "links":{},
+  "description":"Find research and statistics from government",
+  "details":{
+    "document_noun":"statistic",
+    "filter":{
+    },
+    "format_name":"statistic",
+    "show_summaries":true,
+    "sort": [
```

Comment:
I don't think this is correct. From the card:

```
updated, popularity, relevance
(AND release date ordering if user selects 'Upcoming' in Statistics facet)
```

Reply:
Sorry the spec changed and I didn't update the PR. This iteration has been approved by Si+Mia, and we'll be making some further changes later on.